tools/topology/topology2/platform/intel/bt-default.conf

```diff
@@ -1,15 +1,18 @@
 # Default BT variable definitions
 Define {
+	# BT_ID is dai link ID that is matched with machine driver
+	BT_ID			2
+	# BT_INDEX is dai_index which passed to firmware
 	BT_INDEX		2
 	FORMAT			s16le
 	BT_MCLK			38400000
 	BT_NAME			"SSP2-BT"
 
-	BT_PB_HOST_PIPELINE_ID	5
-	BT_PB_DAI_PIPELINE_ID	6
-	BT_PB_HOST_PIPELINE_SINK "copier.host.5.1"
-	BT_PB_DAI_PIPELINE_SRC "copier.SSP.6.1"
-	BT_PB_PIPELINE_STREAM_NAME "copier.SSP.6.1"
+	BT_PB_HOST_PIPELINE_ID	7
+	BT_PB_DAI_PIPELINE_ID	8
+	BT_PB_HOST_PIPELINE_SINK "copier.SSP.8.1"
+	BT_PB_DAI_PIPELINE_SRC "copier.host.7.1"
```

Comment:
The commit message should probably mention these example values were also reversed. This worked with cavs-nocodec as it specified this route explicitly.

Reply:
ok